### DISCUSSION TITLE
JOSM 1.5.x version numbers

### DISCUSSION THREAD

**Discussion by noenandre** [microsoft/winget-pkgs#51082] [Q&A] [answered]
The newest version of "JOSM" is shown as "1.5.18387" on windows both from the exe and the msi installer. But on all other surfaces, like the homepage,  the version number is shown simply as the svn commit number "18387". 
Prior to version 18191 only the commit number was in use. Because people was being offered version 18118 instead of 1.5.18303, (because 18118 is bigger than 1), all older version of Josm was deleted. See #39851
The newest package manifest specification allows one to specify a "DisplayVersion" under "AppsAndFeaturesEntry". Will it be possible and right to add the 1.5.x  version numbers there, and only use the commit number everywhere else?

**Comment by jedieaston** [comment, marked as answer]
That's the plan, although the client can't use those values yet, so we haven't been adding them to the repository.

I would keep an eye on https://github.com/microsoft/winget-cli/issues/1952 as that's what we're waiting on.
